packages/core/src/components/modal/modal.scss

```diff
@@ -122,14 +113,9 @@ $modals: (
   color: var(--tds-modal-text);
   font: var(--tds-body-01);
   letter-spacing: var(--tds-body-01-ls);
-  padding-bottom: 40px;
   overflow-y: visible;
-  padding-right: 16px;
+  padding: 0 16px 16px;
```

Comment:
```suggestion
  padding: 0 var(--tds-spacing-element-16) var(--tds-spacing-element-16);
```

Reply:
@EmelieLitwin @theJohnnyMe @nathalielindqvist @timrombergjakobsson I think is something that should be update in a lot of places. Preferring vars over pixel values